libs/designsystem/src/lib/components/chart/chart-js/chart-js.service.ts

```diff
@@ -1,9 +1,26 @@
 import { ElementRef, Injectable } from '@angular/core';
-import { ActiveElement, ChartConfiguration, ChartOptions } from 'chart.js';
+import {
+  ActiveElement,
+  CartesianScaleOptions,
```

Comment:
I'm receiving a `[tsserver 6133] [I] 'CartesianScaleOptions' is declared but its value is never read` warning on `CartesianScaleOptions`